From aaa7076b52e89b7f45fa53a072f36e0fcf08ca7d Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Tue, 28 Apr 2026 12:01:06 +0200
Subject: [PATCH] acp: partially revert
 f76327e076538b859bf05fff37188e92f3e1493a

The Line Out mute seems to break things.

See #5246

diff --git a/spa/plugins/alsa/mixer/paths/analog-output-headphones.conf b/spa/plugins/alsa/mixer/paths/analog-output-headphones.conf
index 0e698bd9c..3c62c5e67 100644
--- a/spa/plugins/alsa/mixer/paths/analog-output-headphones.conf
+++ b/spa/plugins/alsa/mixer/paths/analog-output-headphones.conf
@@ -93,12 +93,6 @@ volume = merge
 override-map.1 = all
 override-map.2 = all-left,all-right
 
-; Keep Line Out disabled in the headphones path so selecting headphones
-; does not also drive speaker/line outputs on shared controls.
-[Element Line Out]
-switch = off
-volume = off
-
 ; This path is intended to control the first headphones, not
 ; the second headphones. But it should not hurt if we leave the second
 ; headphone jack enabled nonetheless.
diff --git a/spa/plugins/alsa/mixer/paths/analog-output-lineout.conf b/spa/plugins/alsa/mixer/paths/analog-output-lineout.conf
index a6fcfcb83..1ffce2225 100644
--- a/spa/plugins/alsa/mixer/paths/analog-output-lineout.conf
+++ b/spa/plugins/alsa/mixer/paths/analog-output-lineout.conf
@@ -113,23 +113,10 @@ override-map.1 = all
 override-map.2 = all-left,all-right
 required-any = any
 
-[Element Line Out]
-switch = mute
-volume = merge
-override-map.1 = all
-override-map.2 = all-left,all-right
-
 [Element Master Mono]
 switch = off
 volume = off
 
-; Prefer manual routing decisions by disabling codec auto-mute here.
-[Element Auto-Mute Mode]
-enumeration = select
-
-[Option Auto-Mute Mode:Disabled]
-name = analog-output-lineout
-
 [Element Line HP Swap]
 switch = off
 required-any = any

